orchagent/intfsorch.cpp

```diff
@@ -57,6 +57,13 @@ static const vector<sai_router_interface_stat_t> rifStatIds =
     SAI_ROUTER_INTERFACE_STAT_OUT_ERROR_OCTETS,
 };
 
+// Translation of loopback action from sonic to sai type
+const unordered_map<string, sai_packet_action_t> IntfsOrch::m_sai_loopback_action_map =
```

Comment:
You can have it a local map variable.. Also please follow the existing naming convention in the file. No need to have it as part of `IntfsOrch `class

Reply:
Done.